cmd/monogen/testapp/wallet/metamask-integration.js

```diff
@@ -9,9 +9,9 @@ const depositButton = document.getElementById('deposit-button');
 const statusParagraph = document.getElementById('status');
 const amountInput = document.getElementById('amount');
 const recipientInput = document.getElementById('recipient');
-const enableMetamaskButton = document.getElementById('enable-metamask');
+const chainId = "0x384";
```

Comment:
nit: could you re-add the `// 900` comment so the `chainID` is more readable? It's kind of unfortunate that we need to pass through a hex value for chain ID when it's always represented as a base 10 number to the user